source/form.html

```diff
@@ -0,0 +1,168 @@
+<html class="page" lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>Седона</title>
+  </head>
+
+  <body class="page__body">
+
+    <header class="main-header">
+      <a class="main-header__logo">
+        <img src="img/logo.svg" width="185" height="96" alt="логотип">
+      </a>
+      <nav class="main-nav">
+        <button class="main-nav__toggle" type="button"><span class="visually-hidden">Открыть меню</span>
+        </button>
+
+        <div class="main-nav__wrapper">
+          <ul class="main-nav__list site-list">
+            <li class="site-list__item site-list__item--active"><a>Главная</a></li>
+            <li class="site-list__item"><a href="#">Фото и видео</a></li>
+            <li class="site-list__item"><a href="#">Форма отзыва</a></li>
+            <li class="site-list__item"><a href="#">HTML ACADEMY</a></li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+
+
+    <div class="promo">
+      <img src="" alt="" width="901" height="389">
+    </div>
+
+
+
+
+    <article class="form">
```

Comment:
Так как внизу уже есть form, то лучше так:
```suggestion
    <article class="form-page">
```